tests/Microsoft.Bot.Builder.Azure.Tests/BlobStorageBaseTests.cs

```diff
@@ -68,12 +68,12 @@ public async Task TestBlobStorageWriteDeleteRead()
                 var result = await storage.ReadAsync(new[] { "x", "y" });
 
                 // Assert
-                Assert.AreEqual(1, result.Count);
-                Assert.AreEqual("world", result["y"]);
+                Assert.Equal(1, result.Count);
```

Comment:
should this one be `Assert.Single()` ?

Reply:
`Assert.Single` can't be used with Dictionary type, it expects an IEnumerable.